lib/Subscriber/Core/MappingSubscriber.php

```diff
@@ -97,6 +97,9 @@ public function handlePersist(PersistEvent $event)
                 case 'reference':
                     $this->persistReference($node, $accessor, $fieldName, $locale, $fieldMapping);
                     break;
+                case 'json_array':
+                    $this->persistJsonArray($node, $accessor, $fieldName, $locale, $fieldMapping);
+                    break;
```

Comment:
We need to add validation for types, created: https://github.com/sulu-io/sulu-document-manager/issues/68